### ISSUE TITLE
iris activation

### ISSUE THREAD

**Issue by birdspring** [chasingbob/keras-visuals#4]
For iris example, softmax activation function performs better than sigmoid.

**Comment by chasingbob** [comment]
Good find - I have changed to _softmax_, and it is indeed performing better.
